packages/atclient/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(atclient_srcs
  ${CMAKE_CURRENT_LIST_DIR}/src/atstr.c
  ${CMAKE_CURRENT_LIST_DIR}/src/connection.c
  ${CMAKE_CURRENT_LIST_DIR}/src/metadata.c
+ ${CMAKE_CURRENT_LIST_DIR}/src/notification.c
  ${CMAKE_CURRENT_LIST_DIR}/src/stringutils.c
```

Comment:
Other than this, the rest of the changes in this file are to indentation / formatting